dev-client/src/components/home/MapSearch.tsx

```diff
@@ -42,14 +42,46 @@ export default function MapSearch({zoomTo, zoomToUser, toggleMapLayer}: Props) {
   const {t} = useTranslation();
   const [query, setQuery] = useState('');
   const [suggestions, setSuggestions] = useState<Suggestion[]>([]);
+  const [_abortController, setAbortController] =
```

Comment:
nit: if the variable is entirely unused, i think you can avoid naming it at all like this:
```suggestion
  const [, setAbortController] =
```
i've never actually done this, i just have gotten frustrated in the past about how using `_` in JS actually binds a variable named `_`, but i looked it up and apparently that is a thing: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#array_destructuring

Reply:
nice!